Kudu.Core/Deployment/Generator/SiteBuilderFactory.cs

```diff
@@ -194,6 +194,17 @@ private ISiteBuilder ResolveProject(string repositoryRoot, string targetPath, ID
             {
                 return DetermineProject(repositoryRoot, projects[0], perDeploymentSettings, fileFinder);
             }
+            
+            // Check for ASP.NET 5 project without VS solution or project
+            string projectJson;
+            if (AspNet5Helper.TryAspNet5Project(targetPath, out projectJson))
+            {
+                return new AspNet5Builder(_environment,
+                                           perDeploymentSettings,
+                                           _propertyProvider,
+                                           repositoryRoot,
+                                           projectJson);
+            }
 
```

Comment:
@suwatch @regisbsb I do think it makes sense to check ASP.NET 5 before WebsiteProject since an ASP.NET 5 project could include another `csproj` while a normal Website would never include a `project.json`